e2e/scripts/typography.py

```diff
@@ -1,70 +1,6 @@
 import streamlit as st
 
 
-def draw_lorem_ipsum():
```

Comment:
nit: The corresponding snapshots for this now-removed part of the test (`frontend/cypress/snapshots/linux/2x/typography.spec.js/main-block.snap.png` and `frontend/cypress/snapshots/linux/2x/typography.spec.js/main-block-dark.snap.png`) are still around and can be removed